draft-ietf-httpbis-client-cert-field.md

```diff
@@ -142,7 +142,7 @@ the reverse proxy from the reverse proxy to the origin server.
 
 Client-Cert-Chain:
 : Conveys the certificate chain used for validation of the end-entity
-certificate used by the client in the TLS handshake from the reverse proxy to the origin server.
+certificate provided by the client in the TLS handshake with the reverse proxy from the proxy to the origin server.
```

Comment:
This sentence has gotten a little hard to follow.

There are two things here that might need to be separated to make this more readable:

1. This is the chain (less the EE cert) that the client presents in TLS to the TTRP.
2. The TTRP sends this information to the origin server.

Reply:
I know. I know. Saying it is "a little hard to follow" is being too kind. It's pretty terrible. And I seem to make it worse each time I touch it. I'll take another pass over it though with that separation in mind. Maybe leaning more on the text above to make the second point and leaving it out of the Client-Cert/Client-Cert-Chain items.